website/blog/2022-01-12-announcing-oclif-v2.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Announcing oclif v2!
+---
+
+Hello and happy new year! It is our great pleasure to announce that with the turn of a new year we have finally released oclif v2, which uses the new [@oclif/core](https://github.com/oclif/core) library! 🎉
+
+oclif v2 includes many changes that improve the experience of both creating and using an oclif CLI.
+
+Going forward we do not plan to make any updates to oclif v1 and its corresponding libraries except for critical security fixes (please see the [compatibility matrix](#Compatibility-Matrix) for a list of these libraries). We are not completely dropping support yet in order to give developers time to migrate from v1 to v2 but at some point in the future we will archive v1 repositories and deprecate their versions on NPM.
+
+## What’s changing?
+
+### Repository Changes
+
+All repositories under the [oclif org](https://github.com/oclif/) now use `main` as their primary branch and `oclif-v1` as the legacy branch. The `main`  branch corresponds to oclif v2 and the `oclif-v1` branch to oclif v1. If you find any repos that don’t have a `main` branch you can safely assume that they are not used in oclif v2 and will be deprecated at some point in the future.
+
+### Consolidating Tools & Libraries
+
+The [`oclif-dev`](https://github.com/oclif/dev-cli) CLI has now been incorporated into [`oclif`](https://github.com/oclif/oclif). Now you no longer need to install a separate package to manage your entire CLI’s lifecycle.
+
+The following libraries have been consolidated under [`@oclif/core`](https://github.com/oclif/core) and will be deprecated at some point in the future. You can read the [migration guide](https://github.com/oclif/core/blob/main/MIGRATION.md) to see how to update your CLI or plugin to use the new core library.
+
+* `@oclif/command`
+* `@oclif/config`
+* `@oclif/error`
+* `@oclif/help`
+* `@oclif/parser`
+
+### No more single vs multi command CLIs
+
+There is only 1 "type" of oclif v2 CLI but it can have as few or as many commands as a developer wants. This also means there’s only one command for generating CLIs, `oclif generate` as opposed to the old `oclif single` and `oclif multi` commands.
```

Comment:
```suggestion
There's only one "type" of oclif v2 CLI, but it can have as few or as many commands as a developer wants. As a result, there’s only one command to generate CLIs, `oclif generate`, as opposed to the old `oclif single` and `oclif multi` commands.
```